cmd/commands/runtime.go

```diff
@@ -851,13 +856,22 @@ func NewRuntimeUninstallCommand() *cobra.Command {
 	return cmd
 }
 
-func RunRuntimeUninstall(ctx context.Context, opts *RuntimeUninstallOptions) error {
+func createAnalyticsReporter(ctx context.Context) {
+	user, err := cfConfig.GetCurrentContext().GetUser(ctx)
+	// If error, it will default to noop reporter
+	if err != nil {
+		return
```

Comment:
print debug message here, later I will figure out why we can see debug messages.
but you should not swallow the error here.